Locales/German_de.json

```diff
@@ -0,0 +1,864 @@
+	"Practice the DOUBLE SLASH.\n\nHOLD $LMB$ until spark$Tutorial_Slash2$\n\n\nQuickly TAP $LMB$ 2 times!":"Der DOPPELHIEB!\n\n$Tutorial_Slash2$ Halte $LMB$\n bis du den Funken siehst.\n\n\nJetzt zwei Mal schnell $LMB$ drücken!",
+	"Before entering this room charge your sword $Tutorial_Slash2$\n\nand then quickly\nwalk inside and slash!":"Bevor du hier rein gehst, lad deinen Doppelhieb auf,$Tutorial_Slash2$\n\ndann geh schnell rein\nund hau sie um!",
+	"HOLD $RMB$ to shield.\n\nUse mouse to aim shield.":"Halte $RMB$ um abzuwehren.\n\nBenutze die Maus um das Schild in verschiedene Richtungen zu halten.",
+	"Run at wall and hold jump to WALL RUN.\n\nHOLD $KEY_D$$KEY_W$":"Renne auf Wände zu und halte springen um an der Wand zu laufen.\n\nHalte $KEY_D$$KEY_W$",
+	"Knights can destroy wooden blocks, wooden doors and dig dirt. Point at the wall and jab it to get through $LMB$":"Ritter können Holzblöcke, Holztüren und Erde zerstören. Ziel auf die Wand un stich zu um durch zu kommen $LMB$",
+	"Automatically pickup things used by your class.":"Du sammelst Dinge, die für deine Klasse wichtig sind automatisch auf.",
+	"Press $KEY_SPACE$ to light bomb\n\nAim with mouse.\n\nAgain $KEY_SPACE$ throw it.":"Drücke $KEY_SPACE$ um eine Bombe anzuzünden.\n\nZiele mit der Maus.\n\nDrücke $KEY_SPACE$ erneut um die Bombe zu werfen.",
+	"Shield gliding$Tutorial_Glide$\n\n\nJump over the canyon and while in air:\nPoint your cursor upwards\nHOLD $RMB$ for shield.":"Gleiten.$Tutorial_Glide$\n\n\nSpring über die Schlucht. Während du in der Luft bist, halte dein Schild nach oben.\n$RMB$ um das Schild zu bewegen.",
+
+//Tutorial02.as
+	"Stand on the enemy hall$hall$ to capture it.":"Stell dich in die feindliche Basis$hall$ um sie einzunehmen.",
+	"Halls are respawn points and can be used to switch class.":"Du spawnst an Basen. Außerdem kannst du hier deine Klasse ändern.",
+	"In the hall switch class to ARCHER  $ARCHER$\n\nHOLD $KEY_E$ (use key).\n\nHover mouse over the$CLASSCHANGE$button and release\nthe use key.":"Geh in die Basis und werde BOGENSCHÜTZE $ARCHER$.\n\nHalte $KEY_E$.Halte die Maus über $CLASSCHANGE$und lass die Taste los..",
+	"As archer $Tutorial_Archer$\n\n\nAim with mouse.\n\nHOLD$LMB$ to charge\n\nRELEASE to fire arrow!":"Bogenschütze $Tutorial_Archer$\n\n\nZiele mit der Maus.\n\nHalte$LMB$ um den Bogen zu spannen.\n\nLass los um zu feuern!",
+	"Use your grappling hook$Tutorial_Grapple$\n\n\nAim mouse up to catch the ceiling\n\nHOLD$RMB$\n\nSwing to the right $KEY_D$":"Benutz deinen Greifhaken$Tutorial_Grapple$\n\n\nZiele mit der Maus nach oben,\n\ndann drücke $RMB$\n\nSchwing dich rüber $KEY_D$",
+	"Use a combination of grappling hook and wall running to get across here\n\nHOLD$RMB$\n\nHOLD jump & run $KEY_D$$KEY_W$":"Kombinier hier den Greifhaken mit dem Wandlauf.\n\nAn dieser Stelle, Grüße vom Übersetzer: Le Butch.",

```

Comment:
Greetings from the translator: Lebutch. This among other things makes me unhappy with this translation. You've inserted many references to yourself/"the translator" and totally ignored other translations.

Reply:
Sorry for that. I removed all of them. I hope my file is updated now, I am not using github normally...